reference.md

```diff
@@ -0,0 +1,79 @@
+Cheat sheet of functions used in the lessons
+
+
+**Lesson 1**
+
+
+`length ( )` # how many elements are in a particular vector
```

Comment:
I'm being picky, but could you please remove the spaces before the `(` and between the `()` in this function and all others below (e.g. `length()`) as it's the style we use throughout the lesson.